### PR TITLE
Always remove the working directory after creating the capsule archive.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: lib/tool_shed/capsule/capsule_manager.py [hunk 1]
```diff
@@ -78,45 +79,44 @@ def export_repository( self ):
...
+        with self.__tempdir( prefix='tmp-toolshed-export-er' ) as work_dir:
+            try:
+                repositories_archive = tarfile.open( repositories_archive_filename, "w:%s" % self.file_type )
+                exported_repository_registry = ExportedRepositoryRegistry()
+                for repository_id, ordered_repository, ordered_changeset_revision in zip( ordered_repository_ids,
+                                                                                          ordered_repositories,
+                                                                                          ordered_changeset_revisions ):
+                    repository_archive, error_message = self.generate_repository_archive( ordered_repository,
+                                                                                          ordered_changeset_revision,
+                                                                                          work_dir )
```
Comment (nsoranzo) [inline lib/tool_shed/capsule/capsule_manager.py:91]: Is it safe to reuse the same `work_dir` for all the archives? Previously, a new temporary directory was generated for each archive.

Comment (dannon) [inline lib/tool_shed/capsule/capsule_manager.py:91]: @nsoranzo yeah, I saw the same thing.  generate_repository_archive walks the entire work dir it looks like, so they should not be reused.  I proposed a fix in https://github.com/davebx/galaxy/pull/4

**Comment by dannon** [comment]
@galaxybot test this

(can I trigger this directly from jenkins/travis instead of this thing?)

**Comment by martenson** [comment]
@dannon yes, while logged in you can trigger both respectively

**Comment by martenson** [comment]
btw do we want to port this to 16.04? or TS will keep having this problem until 16.07....
@dannon @davebx

**Comment by dannon** [comment]
Well, it isn't an active failure preventing any functionality, it's just a failure to clean up temp files when an export crashes.  We can port, but until we get more disk on toolshed.g2 (or otherwise relocate the temp directory), the number one thing we need to do there is just monitor disk space and keep temp clean.

**Pull request merged**